### PR TITLE
Added Label to User Profile and changed AgreementSignDate to date in Responders Portal.

### PR DESCRIPTION
**Pull request by Christopher-Tihor**

### CODE REVIEW

File: responders/src/API/EMBC.Responders.API/Controllers/ProfileController.cs [hunk 1]
```diff
@@ -166,7 +166,8 @@ public class UserProfile
         public string TeamId { get; set; }
         public string TeamName { get; set; }
         public string Role { get; set; }
-        public bool RequiredToSignAgreement { get; set; }
```
Comment (ytqsl) [inline responders/src/API/EMBC.Responders.API/Controllers/ProfileController.cs:169]: I think this is used by the front end to determine the next route after log in

File: responders/src/API/EMBC.Responders.API/Controllers/ProfileController.cs [hunk 2]
```diff
@@ -185,8 +186,7 @@ public class SecurityMapping : Profile
         public SecurityMapping()
         {
             CreateMap<EMBC.ESS.Shared.Contracts.Team.TeamMember, UserProfile>()
```
Comment (ytqsl) [inline responders/src/API/EMBC.Responders.API/Controllers/ProfileController.cs:188]: Would it make more sense to map teammember to teammember and not user userprofile for updates?

File: responders/src/API/EMBC.Responders.API/Controllers/ProfileController.cs (hunk 1 above)
Comment (Christopher-Tihor) [reply responders/src/API/EMBC.Responders.API/Controllers/ProfileController.cs:169]: Sue requested the change it to a date instead of a boolean.

Comment (ytqsl) [reply responders/src/API/EMBC.Responders.API/Controllers/ProfileController.cs:169]: I'm pretty certain this change will break https://github.com/bcgov/embc-ess-mod/blob/c57b425f1bcfc9ecfe17adca0f499ce6da4086db/responders/src/UI/embc-responder/src/app/app.component.ts#L38

**Review by ytqsl — APPROVED** [review]

**Pull request merged**